pkg/controller/shared_informer.go

```diff
@@ -206,7 +206,7 @@ func (f *deploymentConfigInformer) Informer() framework.SharedIndexInformer {
 		},
 		informerObj,
 		f.defaultResync,
-		cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc},
+		cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc, oscache.ImageStreamReferenceIndex: oscache.ImageStreamReferenceIndexFunc},
```

Comment:
This should probably not be in this package, it should be in `pkg/controller/shared` or something because `pkg/controller` should not know about specific object types.